components/list/list.js

```diff
@@ -193,8 +208,9 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 		if (refresh) {
 			this._itemsShowingTotalCount = await this.getItems().reduce(async(count, item) => {
 				await item.updateComplete;
-				if (item._selectionProvider) {
-					return (await count + await item._selectionProvider._getListItemsShowingTotalCount(true));
+				if (item._hasChildren && (!item.expandable || item.expanded)) {
```

Comment:
Had to swap to using the new `_hasChildren` since `_selectionProvider` is only set when `selectable` is set on `ListItemCheckboxMixin`. So load more component wasn't working properly with non-selectable lists.